app/src/test/java/com/android/unio/model/firestore/HydrationAndSerializationTest.kt

```diff
@@ -48,7 +48,9 @@ class HydrationAndSerializationTest {
           members = User.firestoreReferenceListWith(listOf("1", "2")),
           followersCount = 0,
           image = "https://www.example.com/image.jpg",
-          events = Event.firestoreReferenceListWith(listOf("1", "2")))
+          events = Event.firestoreReferenceListWith(listOf("1", "2")),
+          principalEmailAddress = "example@adress.com",
+          adminUid = "1")
```

Comment:
Consider also adding the assert call in the test, to check if these fields are actually properly serialized and hydrated.

Reply:
Wouldn't it be its own unit test ? As the one I made only check for the validity of the whole process